src/js/models/v2/aboutTheData/CoreReportingRow.js

```diff
@@ -8,25 +8,25 @@ import { format } from 'date-fns';
 
 const CoreReportingRow = {
     populateCore(data) {
-        this._budgetAuthority = data.current_total_budget_authority_amount || 0;
+        this._budgetAuthority = data.current_total_budget_authority_amount;
         this._mostRecentPublicationDate = data.recent_publication_date || null;
         /* eslint-disable camelcase */
-        this._gtasObligationTotal = data.tas_account_discrepancies_totals?.gtas_obligation_total || 0;
-        this._discrepancyCount = data.tas_account_discrepancies_totals?.missing_tas_accounts_count || 0;
+        this._gtasObligationTotal = data.tas_account_discrepancies_totals?.gtas_obligation_total;
```

Comment:
are we reverting this back to their original state?

Reply:
I reverted back the ones that arent dollar amounts.